src/pages/finance/paymentMethod/index.jsx

```diff
@@ -1,69 +1,67 @@
-    Typography
-} from "@mui/material";
-import {Add, CreditCard, LocalAtm} from "@mui/icons-material";
-import {useEffect, useState} from "react";
-import {EmptyState} from "@/components/ui/EmptyState.jsx";
+    AccountBalance,
+    CreditCard,
+    Info,
+    LocalAtm,
+} from "@mui/icons-material";
+import React, {useEffect, useState} from "react";
+
 import {deleteFromApiData, getFromApiData, postToApiData, putToApiData} from "@/api/inesDataApiV1.js";
-import {Link} from "react-router-dom";
-import {grey} from "@mui/material/colors";
 import {ActionModalPM} from "@/pages/finance/paymentMethod/ActionModalPM.jsx";
 import {PaymentMethodsCard} from "@/pages/finance/paymentMethod/PaymentMethodsCard.jsx";
 
-//TODO: add the skeleton
-//TODO: Add filters when the endpoint is able to it
+import {LayoutDataViewList} from "@/layouts/inner/LayoutDataViewList.jsx";
+import {SummaryCard} from "@/components/base/SummaryCard.jsx";
+import {Filters} from "@/components/ui/Filters.jsx";
+import {SortBy} from "@/components/ui/SortBy.jsx";
+import {useSearchParams} from "react-router-dom";
+
+// TODO: Add pagination
 // TODO: last used payment methods on transactions when endpoint is filtering by date
 
 export const PaymentMethodDashboard = () => {
-    const [paymentMethods, setPaymentMethods] = useState([]);
-    const [filters, setFilters] = useState({
-        currencyCode: undefined,
-        name: undefined,
-        type: undefined,
-    });
 
-    const lastPaymentMethods = paymentMethods.length > 0 ? paymentMethods.slice(0, 3) : undefined;
+    const [currentQueryParams, setCurrentQueryParams] = useSearchParams();
+    const searchParams = Object.fromEntries([...currentQueryParams]);
+
+    const [paymentMethods, setPaymentMethods] = useState(undefined)
+    const lastPaymentMethods = paymentMethods && paymentMethods.length > 0 ? paymentMethods.slice(0, 3) : undefined;
 
-    const currenciesAvailable = paymentMethods.length > 0 ? paymentMethods.map(pm => pm.currency) : undefined;
-    const currenciesOnPm = currenciesAvailable && currenciesAvailable.length > 0 ? currenciesAvailable.map(pm => pm.isoCode).filter((value, index, array) => array.indexOf(value) === index) : undefined;
+    const availableCurrencies = [
+        {value: "USD", label: "Dollar"},
+        {value: "EUR", label: "Euro"},
+        {value: "BRL", label: "Real"},
+    ]
 
-    const typesOfPm = paymentMethods.length > 0 ? paymentMethods.map(pm => pm.type).filter((value, index, array) => array.indexOf(value) === index) : undefined;
+    const sortOptions = [
+        {value: "currency", label: "Currency"},
+        {value: "name", label: "Name"},
+        {value: "type", label: "Type"},
+        {value: "createdAt", label: "Created At"},
+    ]
 
     const getPaymentMethods = async () => {
-        const data = await getFromApiData(`paymentmethods`);
+
+        const data = await getFromApiData(`paymentmethods`, searchParams);
         const formatted = data.map((item) => {
```

Comment:
#could: that behaviour could be inside of the get function and if we pass no params it comes as the system default, otherwise it will return in the format that matches the passed param